### ISSUE TITLE
Code Structure

### ISSUE THREAD

**Issue by sodhancha** [neutron0910/codes#1]
Hello Neutron0910,
I loved how you separated code into different folders according to functionality.
If you can build a simple tic-tac toe game by abiding to Android best practices do upload it.

You do not need to use NDK for the game.
Just use simple buttonViews and textViews for this.
[text] [text] [text]
[text] [text] [text]
[text] [text] [text]

Calculate whether player 'X' or 'O' has won onChange of each TextBox.
Code Away 👍 ,

Regards,
Sodhan

**Comment by sodhancha** [comment]
@neutron0910 please reply to this so that I can confirm you got it

**Comment by neutron0910** [comment]
@sodhancha yes sir i got the project , i have to use the button view and the text view and manipulate the textviews in the button with the clicks and finally write a code to follow the rules of the tic-tac-toe game and give a text message whether X or O is the winner .is that it ?

**Comment by sodhancha** [comment]
yes

**Comment by sodhancha** [comment]
Can you finish this by next week ? 14th June and show it to me ?

**Comment by sudarshanAw** [comment]
Yes sir, I will complete the code within a week and show you the code before 14th June .

**Comment by sodhancha** [comment]
ok.

**Comment by sudarshanAw** [comment]
hello sir , i have uploaded the TicTacToe project in Github
please have a look once
https://github.com/sudarshanAw/TicTacToe
and please play it on your device as well
Thank you for your time
